Week1/homework/app.js

```diff
@@ -1,11 +1,129 @@
-'use strict';
 
 {
-  const bookTitles = [
-    // Replace with your own book titles
-    'harry_potter_chamber_secrets',
+  'use strict';
+
+  //  1.1
+
+  const myBooks = [
+    'notes_from_underground',
+    'brave_new_world',
+    'nineteen_eighty_four',
+    'animal_farm',
+    'the_little_prince',
+    'the_alchemist',
+    'essays_montaigne',
+    'blindness',
+    'a_confession',
+    'sherlock_holmes',
   ];
 
-  // Replace with your own code
-  console.log(bookTitles);
+  // 1.2
+  console.log(myBooks);
+
+  //  1.3
+
+  function makeList() {
+    const myBooks = [
+      'notes_from_underground',
+      'brave_new_world',
+      'nineteen_eighty_four',
+      'animal_farm',
+      'the_little_prince',
+      'the_alchemist',
+      'essays_montaigne',
+      'blindness',
+      'a_confession',
+      'sherlock_holmes',
+    ];
+
+    const listDiv = document.createElement('div');
+    document.getElementsByTagName('body')[0].appendChild(listDiv);
+
+    const ulTag = document.createElement('ul');
+    listDiv.appendChild(ulTag);
+
+    for (let i = 0; i < myBooks.length; i++) {
+      const liTag = document.createElement('li');
+      liTag.innerHTML = myBooks[i];
+      ulTag.appendChild(liTag);
+    }
+  }
+
+  // makeList(); 
+  //I canceled function calling to avoid showing on page
+
+
+  const infoBookList = {
+    notes_from_underground: {
+      title: 'Notes from Underground',
+      release: '(1864)',
+      author: 'Dostoyevsky',
+    },
+    brave_new_world: { title: 'Brave New World', release: '(1932)', author: 'Aldous Huxley' },
+    nineteen_eighty_four: { title: '1984', release: '(1949)', author: 'George Orwell' },
+    animal_farm: { title: 'Animal Farm', release: '(1945)', author: 'George Orwell' },
+    the_little_prince: { title: 'The Little Prince', release: '(1943)', author: 'Saint Exupery' },
+    the_alchemist: { title: 'The Alchemist', release: '(1988)', author: 'Paulo Coelho' },
+    essays_montaigne: { title: 'Essays', release: '(1580)', author: 'Montaigne' },
+    blindness_saramago: { title: 'Blindness', release: '(1995)', author: 'Saramago' },
+    a_confession: { title: 'A Confession', release: '(1884)', author: 'Tolstoy' },
+    sherlock_holmes: { title: 'Sherlock Holmes', release: '(1892)', author: 'AC Doyle' },
+  };
+
+
+  function makeBookList() {
+    const divBookList = document.createElement('div');
+    document.getElementsByTagName('body')[0].appendChild(divBookList);
+
+    const ulBookList = document.createElement('ul');
+    divBookList.appendChild(ulBookList);
+
+    const values = Object.values(infoBookList);
```

Comment:
Why do you use `Object.values()`? 
You should have a look ad the message from Wilgert in Slack (https://hackyourfuture.slack.com/archives/CL795TNFQ/p1566576723000500) try to improve you functionality based on this message and if something is not clear, ask a question.